src/vm/assemblynative.cpp

```diff
@@ -250,13 +250,13 @@ void QCALLTYPE AssemblyNative::LoadFromPath(INT_PTR ptrNativeAssemblyLoadContext
         
         // Need to verify that this is a valid CLR assembly. 
         if (!pILImage->CheckILFormat())
-            ThrowHR(COR_E_BADIMAGEFORMAT, BFA_BAD_IL);
+            THROW_BAD_FORMAT(BFA_BAD_IL, &*pILImage);
```

Comment:
This is surely not right, but I can't find my way through the thicket of macros. What's the right way?

Reply:
Never mind, `.GetValue()` seems right.